index.css

```diff
@@ -1,13 +1,131 @@
+:root {
+  font-size: 62.5%;
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-size: 1.6rem;
+}
+
+
 .hero {
   background-image: url("./assets/01_Main_Hero_Mobile.jpg");
   background-size: cover;
   background-position: center 33%;
   height: 100vh;
   width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  display: flex;
+  height: 12rem;
+  justify-content: space-between;
+
+}
+
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  flex: 1;
 }
 
-@media (min-width: 700px) {
+.svg {
+  width: 40rem;
+  flex: 4;
+  
+}
+
+.slogan {
+  display: flex;
+  flex-direction: column;
+  font-family: 'Bebas Neue', cursive;
+  color: whitesmoke;
+  letter-spacing: .2rem;
+  flex: 1;
+  align-items: center;
+  text-align: center;
+  
+}
+
+
+
+button{
```

Comment:
you should not add such specific styling to a generic tag.
Imagine you (or a team colleague) developing further and want to include a button again. If you want to use <button> now, it differs from the standard behaviour and you have to change it in both places.
So it would be bether to add a custom class for this.